README.md

```diff
@@ -0,0 +1,32 @@
+#  Tkey ios example application document
+
+After building complete, you can login via your google account.
+Clicking this button will make account for you, using customAuth sdk.
+
+Once you're logged in, you can run a number of tkey-related functions.
+Buttons allow you to test various modules and tkey built-in functions.
+
+## Main Page
+![image](mainPage.png)
+
+### how to start/Users/janghyeong-gyu/workspace/web3auth/tkey-rust-ios-example/mainPage.png
+
+Once you have the final tkey from initialize and reconstruct tkey, you can test all the features.
+The first time you run Initialize and reconstruct tkey, two shares will be created and the threshold will be set to two.
+This means that both shares will be required for login. (2/2 setting)
+
+### testing on multiple device
+If you want to test logging in with the same google account on different devices, you need to set up additional settings.
+Create an additional security question share by pressing the `Add password` button in the security question module.
+
+After that, try initialize on the second device. If you try it right away, you won't have the necessary shares and fail reconstruction. 
+This is because the threshold required for reconstruct is 2, but the new device only has one existing social login share.
+
+At this point, run `Enter SecurityQuestion password` button to retrieve the security question share which was set on the old device and save it locally to the new device.
+After that, when logging in from that device, you can initialize it directly without entering the security question password.
```

Comment:
I think share transfer should be mentioned here instead.

I don't think the example app has a way to do either of these yet?

Reply:
The feature to import a security question share by entering password was added in #47.
Rather, I don't see any share transfer functionality in the example app right now,
does that mean we need to add it?